### PR TITLE
fix(insights): Add keys to parsed tokens on queries page

### PR DESCRIPTION
**Pull request by 0Calories**
Fixes the unique prop key warning that is always present when viewing the Queries page. This was occuring because the SQL parser was outputting the tokens as a list HTML elements without keys.

image

### CODE REVIEW

**Review by JonasBa — APPROVED** [review]

File: static/app/views/starfish/components/tableCells/spanDescriptionCell.tsx [hunk 1]
```diff
@@ -33,7 +33,12 @@ export function SpanDescriptionCell({
       return rawDescription;
     }
 
-    return formatter.toSimpleMarkup(rawDescription);
+    return (
+      formatter
+        .toSimpleMarkup(rawDescription)
+        // Since the order of the tokens will never change, it's safe to assign the index as the key
+        .map((e, i) => cloneElement(e, {key: i}))
+    );
```
Comment (gggritso) [inline static/app/views/starfish/components/tableCells/spanDescriptionCell.tsx:41]: 🤔 can this code live in `sqlish/formatters/simpleMarkup.tsx` instead? That was every usage will have correct `key=` props

**Review by gggritso — COMMENTED** [review]
Makes sense, thanks! One question about code placement

File: static/app/views/starfish/components/tableCells/spanDescriptionCell.tsx (hunk 1 above)
Comment (0Calories) [reply static/app/views/starfish/components/tableCells/spanDescriptionCell.tsx:41]: I completely missed this file, thanks! I'll move the key logic here

Comment (0Calories) [reply static/app/views/starfish/components/tableCells/spanDescriptionCell.tsx]: Moved it to `simpleMarkup.tsx`, how's it look now?

**Review by gggritso — APPROVED** [review]

File: static/app/views/starfish/components/tableCells/spanDescriptionCell.tsx (hunk 1 above)
Comment (gggritso) [reply static/app/views/starfish/components/tableCells/spanDescriptionCell.tsx]: Nice 👍🏻

**Pull request merged**